google/cloud/aiplatform_v1beta1/services/prediction_service/client.py

```diff
@@ -328,7 +328,7 @@ def explain(
         if endpoint is not None:
             request.endpoint = endpoint
         if instances is not None:
-            request.instances = instances
```

Comment:
Synth will try to revert changes like this in future Autosynth PRs. You can add this entire file to the `exclude` list, or a write a regex replacement in `synth.py`.

Reply:
I had this in `synth.py` but failed to update the relevant line to v1beta1, it has been fixed: https://github.com/googleapis/python-aiplatform/pull/8/files#diff-3620890cb174a7f98c0722895e37bb79R72

Reply:
I can also reprioritize the issues that cause the need for synth hacks. The repeated field one should be trivial.

Reply:
Oh, my bad, it looks like that issue has actually been fixed in a more current generator release.